app/views/reservations/show.html.erb

```diff
@@ -36,74 +36,89 @@
+
+          <% if (is_admin?(current_user)) && @reservation.end_time > DateTime.now %>
+            <div class="grid grid-cols-1 gap-4 place-content-center">
+              <div class="text-center py-2 h-10 font-montserrat font-medium leading-5 border rounded-md text-white bg-blue-umblue hover:underline hover:text-blue-umblue hover:bg-white hover:border-solid hover:border-blue-umblue hover:border-2">
+                <% if @reservation.start_time.to_date == @reservation.end_time.to_date %>
+                  <%= link_to 'Edit Reservation', edit_reservation_path(@reservation), class: "text-base font-medium" if policy(@reservation).edit? %>
+                <% else %>
+                  <%= link_to 'Edit Reservation', edit_long_reservation_path(@reservation), class: "text-base font-medium" if policy(@reservation).edit? %>
+                <% end %>
+              </div>
+              <div class="h-12 text-center font-montserrat font-medium border rounded-md text-blue-umblue bg-yellow-ummaze px-2 hover:underline hover:text-blue-umblue hover:bg-white hover:border-solid hover:border-yellow-ummaze hover:border-2">
+                <%= link_to 'Send Reservation Update', send_reservation_updated_email_path(@reservation), class: "text-base font-medium", data: { turbo_confirm: 'Are you sure?', turbo_method: :get} if policy(@reservation).edit? %>
               </div>
-            <% end %>
-          <% else %>
-            <%= link_to reservation_path(@reservation), data: { turbo_confirm: 'Are you sure?', turbo_method: :delete} do %>
-              <span class="tertiary_button">Cancel Reservation
-                <i class="fa fa-times" aria-hidden="true"></i>
-              </span>
-            <% end %>
+            </div>
           <% end %>
-        <% end %>
-        <div class="my-2">
-          <% if @reservation.approved %>
-            <% unless is_manager?(current_user) %>
-              <% if @reservation.vehicle_report.present? %>
-                <%= link_to 'Vehicle Report', vehicle_report_path(@reservation.vehicle_report), class: "secondary_blue_button" if @reservation.car.present? %>
+
+          <% if (is_student?(current_user) && allow_student_to_edit_reservation?(@reservation) || is_manager?(current_user) && allow_manager_to_edit_reservation?(@reservation)) && !@reservation.approved %>
+            <div class="grid grid-cols-1 gap-4 place-content-center">
+              <% if recurring?(@reservation) %>
+                <%= form_tag cancel_recurring_reservation_url, method: :post, local: true,
+                  data: {
+                      approve_target: "form1"
+                    } do |form| %>
+                  <div class="my-2">
+                    <label for="cancel_type" class="fancy_label">Cancel Recurring Reservation</label>
+                    <%= select_tag :cancel_type, options_for_select(cancel_types), include_blank: "Select to Cancel ...", class: "filter_select",
```

Comment:
Line 91 has a label to Cancel Recurring Reservation in the students' interface. You changed the same label for the admin interface to Cancel Reservations(s). And the Students Cancel button looks like this now:
<img width="717" alt="Screenshot 2023-09-26 at 6 50 46 PM" src="https://github.com/lsa-mis/lsa_ride_share/assets/56089936/8a3161fa-4a7d-449d-9c7c-edd8d5482ec6">

I would keep the word 'recurring ' in this label to make things clear. This we can discuss, but the students's button should be edited anyway.

Reply:
I set the css and syntax to make the two recurring labels equal. 
I thought visually the 'reservation(s)' gave the same message as having the extra word 'recurring'. If you want the word recurring in the label then i will add it back.

Reply:
The labels look good.
About the word 'recurring' in labels. I don't know what is better. There is a label 'Approve All Recurring Reservations'. I am working on 'Update All Recurring Reservations'. I am afraid we'll need something like 'Send All Recurring Update'. Then it makes sense to have 'Cancel Recurring Reservations'. 
Or maybe there should be a section 'Recurring' with all recurring buttons in there.
Should we ask Maria and others about all these buttons and labels?
I don't have a solution that I like.

Reply:
I would have Maria give us her input to the proper naming. I used the '... reservation(s)' to shorten the label

Reply:
I'll show some screenshots tomorrow at the meeting including Update All Recurring Reservations bittons.